scss/_accordion.scss

```diff
@@ -58,39 +56,36 @@
 }
 
 .accordion-item {
+
   &:first-of-type {
+
     .accordion-button {
       @include border-top-radius($accordion-border-radius);
     }
   }
 
+  // Only set a border-radius on the last item if the accordion is collapsed
   &:last-of-type {
```

Comment:
keep the `margin-bottom: 0` will have no impact on our design and keep align with bootstrap